### DISCUSSION TITLE
60 LED / meter WS2812B - only one LED flashing

### DISCUSSION THREAD

**Discussion by mraaro1** [onlaj/Piano-LED-Visualizer#510] [Q&A] [answered]
Hi, I'm having an issue after hooking up the Raspberry Pi to my LED strip. I'm using a 60 LED / m WS2812B, but only the first LED is lighting up, and it's continuously cycling through the different colors, even though Animations are off and no midi signal is being sent. I have an external 5V power brick hooked up to it so it shouldn't be a power issue. 

Is this an issue with using the 60 LED/m strip or something else?

**Comment by onlaj** [comment]
Hello.
Check the wiring. Make sure that the ground (GND) of the Raspberry Pi is connected to the ground of the power supply and the LED strip.

**Reply by mraaro1** [reply]
Thanks! Wiring has been checked and double checked. I'm wondering if there was an issue when I was soldering the pins onto the Raspberry Pi (first time for me), so that the signal is getting crossed. 

Is there an easy way to change the pin from which the data signal is sent from the Raspberry Pi?

**Reply by onlaj** [reply]
You would have to edit this line https://github.com/onlaj/Piano-LED-Visualizer/blob/b51c00b7fe66e102d315e053f927c533dc897e99/lib/ledstrip.py#L31 but I'm not sure what other pin is able to transfer data.

**Comment by mraaro1** [comment, marked as answer]
It turns out the issue was with the solder on my pi. I ended up buying another Zero WH locally and now everything works fine. Thanks for the assistance!

**Comment by bidinou** [comment]
Hi ! I only find those WS2812B strips (144 leds / meter) in 1 meter version, which is probably insufficient for a 88 keys keyboard. Any idea where I could find them ? Longer versions have less density (5 meters versions for instance with 300 leds).

**Reply by onlaj** [reply]
Hey. Ask the seller if they sell them soldered. If not, ask them if they can solder them for you.

**Reply by bidinou** [reply]
Thanks a lot for your reply ! Might be difficult to ask this on amazon though. Oh, would this one be suitable ? https://www.amazon.fr/GOMING-Contr%C3%B4leurs-Individuellement-Adressable-Non-%C3%A9tanche/dp/B0CF57NMHN?source=ps-sl-shoppingads-lpcontext&smid=A3FZWHGYVON1ZP&th=1

2 meters, 160 leds / meter :)

**Reply by onlaj** [reply]
You are paying extra for a controller. But I guess you can use it for a strip's leftovers.

**Reply by bidinou** [reply]
Ah yes indeed ! OK, I found this one : https://www.amazon.fr/Tesfish-Adressable-Individuellement-Intelligente-D%C3%A9coration/dp/B0F9NKWWDD?__mk_fr_FR=%C3%85M%C3%85%C5%BD%C3%95%C3%91&crid=1QO1V38SO7YAA&dib=eyJ2IjoiMSJ9.jsv5HLdSZ_CqM09rG6Xk6lYVtatOy4y8cpXzygCibChEVaCR4WrRRiGXCcHgRgGRexX4uoYr_Ee7nYA75rQyf2JJ--tT4Yb3K7zvo2C5cDczUcqF7-n00qODZwiCTvbY0WVv1y-DAJIREcNGftvs-Wkwud9kplJfHH7ssD7kO4upSmg-QyHQ-m7sJVwIUtuzrbwaWA1ncZEjyk3o501eRcFojByRqVKUE-Vqr14dieg5Tz0pdf6ALZv1kdTZwDKaFrd7VLkwYvZwwNPJTHptIzAl53vZPWsdt811RroTpJs.20djW1hZaP-b1OWhXCrBR-0AmQcCX0_7nWrElrzh7zE&dib_tag=se&keywords=Ws2812B&qid=1765041021&sprefix=ws2812b%2Caps%2C95&sr=8-12&th=1 
:-) 
Unfortunately, couldn't find one from a smaller / local shop with such density.
Thanks a lot for your help !
